src/components/RatingsChart/RatingsChart.tsx

```diff
@@ -235,6 +235,9 @@ export class RatingsChart extends React.Component<RatingsChartProperties, Rating
     }
 
     setRanges = () => {
+        if (!this.container) {
```

Comment:
Should this be `!this.container.current` ?

Reply:
Ah - I think that probably isn't needed anymore, once I got rid of `this.container = null` from `deinitialize`.

I'll take it out and check things out.